src/frontend/src/views/Upload/components/Metadata/components/StaticTable/components/Row/index.tsx

```diff
@@ -26,21 +34,53 @@ export default memo(function Row({ id, metadata }: Props): JSX.Element {
     publicId,
   } = metadata;
 
+  const validatedCellData: Record<
+    string,
+    React.ReactElement | string | undefined
+  > = {
+    privateId: privateId || "--",
```

Comment:
`--` should probably be a constant (in fact, I think it might already be one somewhere in the codebase) that can be reused